### ISSUE TITLE
RuntimeError with sslv2

### ISSUE THREAD

**Issue by jsf9k** [nabla-c0d3/sslyze#319]
As you can see below, I am consistently getting a `RuntimeError - A socket was already set` message when running this command:
```
$ sslyze --sslv2 --starttls=smtp mail1.nrc.gov:25

 AVAILABLE PLUGINS
 -----------------
  SessionRenegotiationPlugin
  HeartbleedPlugin
  FallbackScsvPlugin
  CompressionPlugin
  HttpHeadersPlugin
  RobotPlugin
  SessionResumptionPlugin
  CertificateInfoPlugin
  OpenSslCipherSuitesPlugin
  OpenSslCcsInjectionPlugin

 CHECKING HOST(S) AVAILABILITY
 -----------------------------
   mail1.nrc.gov:25                        => 148.184.176.41 

 SCAN RESULTS FOR MAIL1.NRC.GOV:25 - 148.184.176.41
 --------------------------------------------------
 * SSLV2 Cipher Suites:
     Undefined - An unexpected error happened:
        SSL_CK_RC4_128_WITH_MD5                           RuntimeError - A socket was already set                     
        SSL_CK_RC4_128_EXPORT40_WITH_MD5                  RuntimeError - A socket was already set                     
        SSL_CK_RC2_128_CBC_WITH_MD5                       RuntimeError - A socket was already set                     
        SSL_CK_RC2_128_CBC_EXPORT40_WITH_MD5              RuntimeError - A socket was already set                     
        SSL_CK_IDEA_128_CBC_WITH_MD5                      RuntimeError - A socket was already set                     
        SSL_CK_DES_64_CBC_WITH_MD5                        RuntimeError - A socket was already set                     
        SSL_CK_DES_192_EDE3_CBC_WITH_MD5                  RuntimeError - A socket was already set                     


 SCAN COMPLETED IN 6.55 S
 ------------------------
```

Is this expected?  I'm using the latest `sslyze` code in the `master` branch and the latest `nassl` from PyPI.

**Comment by swarooppatra** [comment]
I am also observing same issue. Below is snippet of output. Can I consider this as below ciphers are not accepted.

* SSLV2 Cipher Suites:
      Server rejected all cipher suites.
     Undefined - An unexpected error happened:
        SSL_CK_RC4_128_WITH_MD5                           RuntimeError - A socket was already set
        SSL_CK_RC4_128_EXPORT40_WITH_MD5                  RuntimeError - A socket was already set
        SSL_CK_RC2_128_CBC_WITH_MD5                       RuntimeError - A socket was already set
        SSL_CK_RC2_128_CBC_EXPORT40_WITH_MD5              RuntimeError - A socket was already set
        SSL_CK_IDEA_128_CBC_WITH_MD5                      RuntimeError - A socket was already set
        SSL_CK_DES_64_CBC_WITH_MD5                        RuntimeError - A socket was already set
        SSL_CK_DES_192_EDE3_CBC_WITH_MD5                  RuntimeError - A socket was already set

Thanks,
Swaroop

**Comment by jsf9k** [comment]
@swarooppatra, I don't think it means the ciphers are not accepted necessarily.  It could be an internal error, but I haven't been able to track down the exact cause yet.

**Comment by swarooppatra** [comment]
Thanks @jsf9k.

**Comment by nabla-c0d3** [comment]
Fixed in https://github.com/nabla-c0d3/sslyze/commit/eeffed5b4c70a152b9bc5d8e966ace088dba1f2a

**Comment by jsf9k** [comment]
Thanks @nabla-c0d3!

**Comment by swarooppatra** [comment]
Thanks @nabla-c0d3 . Can you please let me know which version of sslyze will have this feature. I just updated to 1.4.3 and I see same error. Thanks for this tool.

**Comment by nabla-c0d3** [comment]
It will be released with v2.0.0.

**Comment by nabla-c0d3** [comment]
Released as part of [2.0.0](https://github.com/nabla-c0d3/sslyze/releases/tag/2.0.0).
